### ISSUE TITLE
[RPC][transform] Convert transform sub-commands to first-class commands

### ISSUE THREAD

**Issue by tkotosz** [phpactor/phpactor#1010]
Possible sub-commands of `transform` at the moment:
- complete_constructor
- add_missing_properties
- fix_namespace_class_name
- implement_contracts

I think it would be good to convert these "sub-command" to "first-class" commands because:
- it would allow to run them directly without going through the `transform` command
- it would allow to specify extra arguments for any of them without affecting the others if the need arises later
- it would allow to list them in the context menu directly: currently you need to pick transform from the context menu, then pick a transformation, instead of just picking the command in 1 step (unless you provide a key-binding to trigger the transform-choice menu directly)
- it would allow to show/hide them in the context menu based on the given context: e.g. in the constructor the `complete_constructor` is relevant while the `fix_namespace_class_name` and `implement_contracts` are not
- it would simplify the context menu: having to go multiple choice steps can be annoying

Basically what I am saying is that it is better to make the `context_menu` command a bit more smart and let it provide relevant commands in the given context instead of providing commands which trigger further choice menus (triggering an input callback is still fine if the picked command needs further input).

Note that even if you convert these to "first-class" command as I suggested above you could still keep the current "transform choice menu" if you want to. It would be just a chooser type command like the `context_menu` anyway.

**Comment by dantleech** [comment]
These are "context less" refactorings - they apply to the entire file but there is no reason they cannot be part of the context menu - https://github.com/phpactor/phpactor/blob/develop/lib/Extension/ContextMenu/menu.json (other than running out of space and key bindings ...)
